templates/certified/model-details.html

```diff
@@ -1,6 +1,6 @@
 {% extends "templates/base.html" %}
 
-{% block title %} {{ vendor }} {{ model }} certified with Ubuntu{% endblock %}
+{% block title %} {{ vendor }} {{ model }} {{ form_factor }} certified with Ubuntu{% endblock %}
```

Comment:
The [issue](https://github.com/canonical/ubuntu.com/issues/11826) asks that the page title be:
>Lenovo Think Centre Neo30a 22 certified with Ubuntu

but the demo shows:
>Lenovo  all-in-one certified with Ubuntu

Looking at the `h1` on the page, I think the tags should be:
```suggestion
{% block title %} {{ vendor }} {{ name }} certified with Ubuntu{% endblock %}
```

Reply:
@carkod maybe I was wrong, the page is 500ing now.

Reply:
Right, thanks.